source/more-info/unhealthy/setup.markdown

```diff
@@ -0,0 +1,24 @@
+---
+title: "Setup of the Supervisor failed"
+description: "More information on why failing the setup stage makes a installation as unhealthy."
+---
+
+## The issue
+
+This happens when any of the setup tasks fails to complete, this can be due to the host not being completly ready when the Supervisor starts or that [DBUS] is not properly working.
+
+## The solution
+
+If the issue is related to DBUS, you will see an unsupported message about that as well, you can have a look here [DBUS] on how to resolve that.
+
+The first thing you should try is to restart the Supervisor.
+This can be done from the "System" tab in the Supervisor panel, on the card for "Supervisor" there is a button to restart the Supervisor.
```

Comment:
```suggestion
The first thing you should try is to restart the Supervisor.
This can be done from the "System" tab in the Supervisor panel. On the card for "Supervisor", there is a button to restart the Supervisor.
```